components/producers/producer_test.go

```diff
@@ -75,9 +75,9 @@ func TestWriteDraconOut(t *testing.T) {
 	require.NoError(t, proto.Unmarshal(pBytes, &res))
 
 	assert.Equal(t, "dracon-test", res.GetToolName())
-	assert.Equal(t, "./foobar", res.GetIssues()[0].GetTarget())
-	assert.Equal(t, "./barfoo", res.GetIssues()[0].GetTitle())
-	assert.Equal(t, "./example.yaml", res.GetIssues()[0].GetDescription())
+	assert.Equal(t, "foobar", res.GetIssues()[0].GetTarget())
```

Comment:
Can we `require.NotEmpty(t, res.GetIssues())` to avoid potential panics from accessing `res.GetIssues()[0]`?

It would also help using an helper var to store the first issue :)